test/system/030-run.bats

```diff
@@ -608,6 +608,18 @@ json-file | f
     # a subdir of a volume.
     run_podman run --rm --workdir /IamNotOntheImage -v $testdir/content:/IamNotOntheImage/foo $IMAGE cat foo
     is "$output" "$randomcontent" "cat random content"
+
+    # Make sure that running on a read-only rootfs works (#9230).
+    if ! is_rootless && ! is_remote; then
+    	# image mount is hard to test as a rootless user
```

Comment:
There's a mix of spaces and tabs on this line.

Reply:
Need to fix my vimrc. Thank you!